util/src/main/java/com/redhat/lightblue/util/ServoGraphiteSetup.java

```diff
@@ -68,19 +71,28 @@ public static void initialize() {
         }
     }
 
-    private static synchronized void doInitialize() {
-        if (initialized) {
+    /**
+     * If there is sufficient configuration, register a Graphite observer to
+     * publish metrics. Requires at a minimum environment variable
+     * GRAPHITE_HOSTNAME. Optionally can set GRAPHITE_PREFIX as well as
+     * GRAPHITE_PORT. GRAPHITE_PREFIX defaults to the hostname and GRAPHITE_PORT
+     * defaults to '2004'.
+     */
+    private static void registerGraphiteMetricObserver(List<MetricObserver> observers) {
+        String prefix = System.getenv("GRAPHITE_PREFIX");
+        String host = System.getenv("GRAPHITE_HOSTNAME");
+        String port = System.getenv("GRAPHITE_PORT");
+
+        // verify at least hostname is set, else cannot configure this observer
+        if (null == host || host.trim().isEmpty()) {
+            LOGGER.info("GraphiteMetricObserver not configured, missing environment variable: GRAPHITE_HOSTNAME");
             return;
         }
 
-        // register hystrix servo metrics publisher
-        HystrixPlugins.getInstance().registerMetricsPublisher(HystrixServoMetricsPublisher.getInstance());
```

Comment:
I think it would be good to keep this line if it can help us in case we made any change that would use Hystrix command prior than this execution (as mentioned in the netflix issue)